src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,578 @@
+      <description>RankedVideoStreamPrioritiesList</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0017" side="server" define="SOFT_RECORDING_PRIVACY_MODE_SETTING" type="boolean" writable="true" optional="true">
+      <description>SoftRecordingPrivacyModeSetting</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0018" side="server" define="SOFT_LIVESTREAM_PRIVACY_MODE_SETTING" type="boolean" writable="true" optional="true">
+      <description>SoftLivestreamPrivacyModeSetting</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0019" side="server" define="HARD_PRIVACY_MODE" type="boolean" reportable="true" optional="true">
+      <description>HardPrivacyMode</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001A" side="server" define="NIGHT_VISION" type="TriStateAuto" min="0x00" max="0x02" writable="true" optional="true">
+      <description>NightVision</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001B" side="server" define="NIGHT_VISION_ILLUM" type="TriStateAuto" min="0x00" max="0x02" writable="true" optional="true">
+      <description>NightVisionIllum</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001C" side="server" define="AWB" type="boolean" writable="true" optional="true">
+      <description>AWB</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001D" side="server" define="SHUTTER_SPEED" type="boolean" writable="true" optional="true">
+      <description>ShutterSpeed</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001E" side="server" define="ISO" type="boolean" writable="true" optional="true">
+      <description>ISO</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001F" side="server" define="VIEWPORT" type="ViewportStruct" optional="true">
+      <description>Viewport</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0020" side="server" define="SPKR_ON_OFF" type="boolean" writable="true" optional="true">
+      <description>SpkrOnOff</description>
```

Comment:
Spec comment: here and for the other speaker attributes, should just spell out "Speaker" instead of the weird abbreviation.

Reply:
Expanded to Speaker